src/wap_client.c

```diff
@@ -211,7 +221,8 @@ signal_have_save_ok (client_t *self)
 static void
 signal_have_start_ok (client_t *self)
 {
-    zsock_send (self->cmdpipe, "si", "START OK", 0);
+    zsock_send (self->cmdpipe, "sii", "START OK", 0,
+                wap_proto_curr_height (self->message));
 }
```

Comment:
This is hand written. All generated code has a large notice at the start.
The generated files for the client API are wap_client_engine.inc and
wap_client.h.

On Sun, Jan 4, 2015 at 4:27 PM, oranjuice <notifications@github.com> wrote:

> In src/wap_client.c
> <https://github.com/oranjuice/monero-0mq/pull/4#discussion-diff-22439999>:
>
> > @@ -211,7 +221,8 @@ signal_have_save_ok (client_t *self)
> >  static void
> >  signal_have_start_ok (client_t *self)
> >  {
> > -    zsock_send (self->cmdpipe, "si", "START OK", 0);
> > +    zsock_send (self->cmdpipe, "sii", "START OK", 0,
> > +                wap_proto_curr_height (self->message));
> >  }
>
> was this generated code? This change doesn't seem to appear for me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oranjuice/monero-0mq/pull/4/files#r22439999>.
>